Trappist/src/Promact.Trappist.Repository/TestConduct/TestConductRepository.cs

```diff
@@ -41,6 +41,7 @@ public TestConductRepository(TrappistDbContext dbContext, ITestsRepository testR
             _questionRepository = questionRepository;
             _configuration = configuration;
             _stringConstants = stringConstants;
+            client = new HttpClient();
```

Comment:
you need to create http client wrapper which can be mocked in unit tests, dont use httpclient directly